packs/feats/mind-projectiles.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You have learned to stretch your mind's influence further, releasing projectiles with a swing of your mind weapon. You can make ranged mind weapon Strikes; these are ranged Strikes with a maximum range of 30 feet that deal [[/r 1d6]]{1d6} damage of the same type as your mind weapon. Your ranged mind weapon Strike gains all the benefits of your mind weapon's runes as long as they still apply to a ranged weapon. For example, if your weapon had <em>+1</em>, <em>striking</em>, and <em>spell storing</em> runes, you would get a +1 item bonus to hit with your ranged mind weapon Strike, and it would deal the additional damage from the <em>striking</em> rune, but it wouldn't be able to unleash a spell from the <em>spell storing</em> rune, as that rune can be etched onto only melee weapons.</p>"
+            "value": "<p>You have learned to stretch your mind's influence further, releasing projectiles with a swing of your mind weapon. You can make ranged mind weapon Strikes; these are ranged Strikes with a maximum range of 30 feet that deal @Damage[1d6]{1d6} damage of the same type as your mind weapon. Your ranged mind weapon Strike gains all the benefits of your mind weapon's runes as long as they still apply to a ranged weapon. For example, if your weapon had <em>+1</em>, <em>striking</em>, and <em>spell storing</em> runes, you would get a +1 item bonus to hit with your ranged mind weapon Strike, and it would deal the additional damage from the <em>striking</em> rune, but it wouldn't be able to unleash a spell from the <em>spell storing</em> rune, as that rune can be etched onto only melee weapons.</p>"
```

Comment:
```suggestion
            "value": "<p>You have learned to stretch your mind's influence further, releasing projectiles with a swing of your mind weapon. You can make ranged mind weapon Strikes; these are ranged Strikes with a maximum range of 30 feet that deal @Damage[1d6[untyped]] damage of the same type as your mind weapon. Your ranged mind weapon Strike gains all the benefits of your mind weapon's runes as long as they still apply to a ranged weapon. For example, if your weapon had <em>+1</em>, <em>striking</em>, and <em>spell storing</em> runes, you would get a +1 item bonus to hit with your ranged mind weapon Strike, and it would deal the additional damage from the <em>striking</em> rune, but it wouldn't be able to unleash a spell from the <em>spell storing</em> rune, as that rune can be etched onto only melee weapons.</p>"
```